medicines/doc-index-updater/stunnel.conf

```diff
@@ -4,4 +4,4 @@ foreground = yes
 [doc_index_updater]
 client = yes
 accept = 127.0.0.1:6379
-connect = doc-index-updater-non-prod.redis.cache.windows.net:6380
+connect = doc-index-updater-dev.redis.cache.windows.net:6380
```

Comment:
did you mean to commit this? I want it in master anyway because I've had to do it locally too, so I don't mind

or… we could get this to read from `.env`?

Reply:
Can we? We don't have any code here AFAIK.